### PR TITLE
archwiki-rs: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/archwiki-rs.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# archwiki-rs
+
+> Lee, busca y descarga páginas de la ArchWiki.
+> Más información: <https://gitlab.com/lucifayr/archwiki-rs>.
+
+- Lee una página de la ArchWiki:
+
+`archwiki-rs read-page {{título_página}}`
```
Comment (tricantivu) [inline pages.es/common/archwiki-rs.md:8]:
```suggestion
`archwiki-rs read-page {{título_de_página}}`
```

File: pages.es/common/archwiki-rs.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Lee una página de la ArchWiki en el formato especificado:
+
+`archwiki-rs read-page {{título_página}} --format {{plain-text|markdown|html}}`
```
Comment (tricantivu) [inline pages.es/common/archwiki-rs.md:12]:
```suggestion
`archwiki-rs read-page {{título_de_página}} --format {{plain-text|markdown|html}}`
```

File: pages.es/common/archwiki-rs.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Busca en ArchWiki páginas que contengan el texto proporcionado:
```
Comment (tricantivu) [inline pages.es/common/archwiki-rs.md:14]:
```suggestion
- Busca páginas en ArchWiki con el texto proporcionado:
```

File: pages.es/common/archwiki-rs.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`archwiki-rs search " {{texto_buscado}}" --text-search`
```
Comment (tricantivu) [inline pages.es/common/archwiki-rs.md:16]:
```suggestion
`archwiki-rs search "{{texto_a_buscar}}" --text-search`
```

Is the space after the initial double quote needed?

**Review by tricantivu — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

**Pull request merged**